Makefile

```
@@ -0,0 +1,21 @@
+##@ Makefile
+
+##@ Define the default version
+VERSION ?= latest
```

```suggestion
##@ Define the default airbyte-ci version
AIRBYTE_CI_VERSION ?= latest
```

As this is the root Makefile I think its better to mention that the version is the airbyte-ci one.